_docs/services/intro.md

```diff
@@ -73,7 +73,7 @@ To list all the managed services and plans available to a given space, you run `
     <li><strong>Beta</strong>: The service is stable but still requires further development to ensure it can be deployed to production systems.</li>
     <li><strong>Alpha</strong>: The service is under development and some downtime or data loss can occur.</li>
     <li><strong>Deprecated</strong>: The service is deprecated and will be removed in a future release - please update.</li>
-
+    <li><strong>Legacy</strong>: The service is deprecated, and has been superseded by a preferred fully-supported service.<li>
```

Comment:
```suggestion
    <li><strong>Legacy</strong>: The service is deprecated, and has been superseded by a preferred fully-supported service.</li>
```

The unclosed <li> is causing issues in the `Extending the marketplace` output:

https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.sites.pages.cloud.gov/preview/cloud-gov/cg-site/peterb/fix-deprecated/docs/services/intro/